expertsystem/io/dot.py

```diff
@@ -30,55 +32,88 @@ def write(instance: object, filename: str) -> None:
         stream.write(output_str)
 
 
-_DOT_HEAD = """
-digraph {
+_DOT_HEAD = """digraph {
     rankdir=LR;
-    node [shape=point];
-    edge [arrowhead=none, labelfloat=true];
-    """
+    node [shape=point, width=0];
+    edge [arrowhead=none];
+"""
 _DOT_TAIL = "}\n"
 _DOT_RANK_SAME = "    {{ rank=same {} }};\n"
 _DOT_DEFAULT_NODE = '    "{}" [shape=none, label="{}"];\n'
 _DOT_DEFAULT_EDGE = '    "{}" -> "{}";\n'
 _DOT_LABEL_EDGE = '    "{}" -> "{}" [label="{}"];\n'
 
 
-def __graph_to_dot(graph: StateTransitionGraph) -> str:
-    def node_name(edge_id: int, node_id: Optional[int] = None) -> str:
-        if node_id is None:
-            return f"edge{edge_id}"
-        return f"node{node_id}"
+def __graph_list_to_dot(graphs: List[StateTransitionGraph]) -> str:
+    dot_source = _DOT_HEAD
+    for i, graph in enumerate(graphs):
+        dot_source += __graph_to_dot_content(graph, prefix=f"g{i}_")
```

Comment:
just call the proposed `__graph_to_dot` function

Reply:
That's not how DOT syntax works

Reply:
Ok, sorry I misunderstood the code then. So to clarify, the head and tail is only required for one single dot file, not for each graph?